packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: Client-based rate limits
+title: Client-based rate limits reference
 excerpt: >-
   Understand what client-based rate limits at Okta and learn how to design for efficient use of resources
 ---
 
 # Client-based rate limits
 
- Client-based rate limiting applies to unauthenticated endpoints within Okta that are used during the client application access process. This applies to two types of client access:
+ Client-based rate limiting applies to unauthenticated endpoints within Okta that are used during the access process for client applications. This applies to two types of client access:
 
 * A client accessing an OAuth 2.0 application, which uses a combination of client ID, IP address, and device identifier for rate-limiting.
 
 * A client accessing a non-OAuth 2.0 application (as is the case for a Classic Engine org `/login/login.htm` endpoint), which uses the IP address and device identifier for rate-limiting.
 
-The client access process can include requests to multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine.
+The client access process can include requests to multiple API endpoints that differ based on your type of org: Okta Identity Engine or Okta Classic Engine. The Identity Engine endpoints can include `/idp/idx`, `/idp/idx/identify`, `/idp/idx/introspect`, and [Identity Engine App intent links](/docs/concepts/oie-intro/#app-intent-links).
 
-The Classic Engine and the Identity Engine both include client-based rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint provides granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint, and this framework isolates rogue OAuth 2.0 clients and bad actors, thereby ensuring that valid users and applications don't run into rate limit violations.
+The Classic Engine and the Identity Engine both include client-based rate-limiting for the OAuth 2.0 API endpoint `/authorize`. This endpoint provides granular isolation for requests made to the OAuth 2.0 `/authorize` endpoint, and this framework isolates rogue OAuth 2.0 clients and bad actors, ensuring that valid users and applications don't run into rate limit violations.
```

Comment:
The 2nd sentence in this para is quite long. I wonder if we could break it up a bit?

Reply:
I find breaking up compound sentences to reduce paragraph/sentence length, which I know Acrolinx is fond of, introduces a staccato cadence that is somewhat grating, in my opinion. I trust Acrolinx would not like that sentence either. It could be recast, but I'd need to spend more time with the content to make sure it remains accurate. For the time being, I'm going to leave as is.